vaccine_feed_ingest/runners/nc/myspot_gov/normalize.py

```diff
@@ -43,27 +45,42 @@ def _get_location(site: dict):
 def _get_contacts(site: dict):
     ret = []
     if site["Appointment Phone"] != "":
-        raw_phone = str(site["Appointment Phone"]).lstrip("1")
-        if raw_phone[3] == "-" or raw_phone[7] == "-":
-            phone = "(" + raw_phone[0:3] + ") " + raw_phone[4:7] + "-" + raw_phone[8:12]
-            phone_notes = raw_phone[12:]
-        elif len(raw_phone) == 10:
-            phone = "(" + raw_phone[0:3] + ") " + raw_phone[3:6] + "-" + raw_phone[6:10]
-            phone_notes = ""
+        ret.extend(_parse_phone_numbers(site["Appointment Phone"]))
+
+    url = site["Web Address"]
+    # Some URLs have multiple schemes.
+    valid_url = re.match(r"(https?:\/\[/)*:(.+)", url)
+    if valid_url is not None:
+        if valid_url.group(1) is None:
+            url = valid_url.group(2)
         else:
-            phone = raw_phone[0:14]
-            phone_notes = raw_phone[14:]
+            url = f"{valid_url.group(1)}{valid_url.group(2)}"
 
-        if phone_notes == "":
-            ret.append(schema.Contact(phone=phone))
-        else:
-            phone_notes = phone_notes.lstrip(",")
-            phone_notes = phone_notes.lstrip(";")
-            phone_notes = phone_notes.lstrip(" ")
-            ret.append(schema.Contact(phone=phone, other=f"phone_notes:{phone_notes}"))
+        url = normalize_url(url)
+        ret.append(schema.Contact(website=url))
+    return ret
+
+
+phone_number_parser = re.compile(
+    r"^(?:(?:\+?1\s*(?:[.-]\s*)?)?(?:\(?([0-9]{3})\)?)\s*(?:[.-]\s*)?)?([0-9]{3})\s*(?:[.-]\s*)?([0-9]{4})\s*(?:\([A-Z]{4}\))?,?(?:\s*(?:\#|x\.?|ext\.?|extension|OPTION|Option|option|Ext)\s*(\d+))?"  # noqa: E501
+)
```

Comment:
Can we potentially use the regex directly from `vaccine_feed_ingest_schema` by importing it (or is this a _different_ regex)?

Reply:
It's a different regex :(

In particular, it is much more flexible about formatting, since I'm trying to parse rather than validate.